meta.yml

```diff
@@ -1,13 +1,13 @@
 fullname: Apery
-shortname: mathcomp-apery
+shortname: apery
 organization: math-comp
+opam_name: coq-mathcomp-apery
 community: false
-travis: true
+action: true
```

Comment:
Ha, I guess the answer to my previous comment about the mysterious diff is here. So on which basis should I choose between `action` and `travis`?

Reply:
Travis started severely limiting (on a per-organization basis) the amount of computation time given to non-paying open-source projects. Unless you have a strong reason to use Travis for one of its features, don't.